docs/docs/addons-pure-render-mixin.md

```diff
@@ -1,17 +1,31 @@
 ---
 id: pure-render-mixin
-title: PureRenderMixin
-permalink: docs-old/pure-render-mixin.html
-prev: update.html
-next: perf.html
+title: PureRenderMixin (Deprecated)
+layout: docs
+category: Add-Ons
+permalink: docs/pure-render-mixin.html
+prev: clone-with-props.html
+next: two-way-binding-helpers.html
 ---
 
-If your React component's render function is "pure" (in other words, it renders the same result given the same props and state), you can use this mixin for a performance boost in some cases.
+> Note: PureRenderMixin is deprecated.
```

Comment:
It is not deprecated but legacy. This is an important distinction because deprecated APIs get killed in next version but legacy APIs live on, even if discouraged.